win_installer/cppcheck.wxs

```diff
@@ -93,12 +81,16 @@
               <File Id='boost.cfg' Name='boost.cfg' Source='$(var.CfgsDir)\boost.cfg' />
               <File Id='bsd.cfg' Name='bsd.cfg' Source='$(var.CfgsDir)\bsd.cfg' />
               <File Id='cairo.cfg' Name='cairo.cfg' Source='$(var.CfgsDir)\cairo.cfg' />
+              <File Id='cppcheck-lib.cfg' Name='cppcheck-lib.cfg' Source='$(var.CfgsDir)\cppcheck-lib.cfg' />
```

Comment:
cppcheck users likely do not need this cppcheck-lib.cfg file.

Reply:
That's possibly true for most of the files. 😁

Since we offer an API (and the accompanied `cppcheck-core.dll`) I think it makes sense to include this. I also assume the Linux packages do not omit this.

Reply:
But this cppcheck-lib.cfg is only intended for our self check. I do not believe any users use cppcheck-core.dll in their software.

Reply:
> Since we offer an API 

I do not know that we offer an API so the cppcheck-core.dll can be used. In theory you can copy the headers from our lib folder.. however those are changed every single version they are highly unstable.

Reply:
This is included in the linux packages since `make install` copies all of them unconditionally: https://fedora.pkgs.org/38/fedora-aarch64/cppcheck-2.9-4.fc38.aarch64.rpm.html.

We need tests for the built packages in the CI which checks against a filelist for missing/unexpected files.